src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -835,6 +838,42 @@ pub enum RoundMode {
 }
 
 impl Decimal {
+    /// abs the Decimal into a new Decimal.
+    #[inline]
+    pub fn abs(&self) -> Res<Decimal> {
+        let mut res = self.clone();
+        res.negative = false;
+        Res::Ok(res)
+    }
+
+    /// ceil the Decimal into a new Decimal.
+    pub fn ceil(&self) -> Res<Decimal> {
+        let mut target = if self.frac_cnt > 0 && !self.negative {
```

Comment:
Why not use `t.round(0, RoundMode::Ceiling)` directly?

Reply:
Can we fix the bug in `t.round(0, RoundMode::Ceiling)` directly?

Reply:
Can we fix the bug in `t.round(0, RoundMode::Ceiling)` directly?

Reply:
Can we fix the bug in t.round(0, RoundMode::Ceiling) directly? @hicqu

Reply:
`round` is used many place, and it's different from `math.Ceil` semantically. I prefer use different `RoundMode` name, such as `RoundUp` and `RoundDown`. They are more clearly. If it's OK, I will do it in next PR.